src/app/core/app.component.ts

```diff
@@ -60,6 +61,8 @@ export class AppComponent implements OnInit, OnDestroy {
     if (appConfig.theme !== 'default') {
       this.renderer.setAttribute(this.el.nativeElement, 'data-theme', `${appConfig.theme}`);
     }
+    // eslint-disable-next-line no-console
+    console.log(`App version: ${version}`);
```

Comment:
Angular keeps its version as an html attribute `ng-version="13.1.2"`, we could do the same with `alg-version=...`

Reply:
What for? Instead of printing it? 
The print was a kind of easy way to ask the customer for the version while we do something better with it in a future PR (e.g. display it somewhere in small).

Reply:
Because sometimes when you have to many logs it might be drowned in the logs, or one might clear logs which makes retrieving the version no longer possible, scenarios like that.
Having it as an attribute on `alg-root` or `html` makes it readable at first glance on the element panel, no matter how long you've been browsing the app.
Anyway it was only a suggestion, if there's still no interest in your opinion it's fine by me.